src/controllers/index.js

```diff
@@ -1,24 +1,39 @@
-const express = require('express');
-const queries = require('../queries');
+const express = require("express");
+const queries = require("../queries");
+const bodyParser = require("body-parser");
 const router = express.Router();
-const path = require('path');
+const path = require("path");
+// var sqlinjection = require("sql-injection");
 
-router.get('/getList', (req,res) => {
-    var list = ["item1", "item2", "item3"];
-    res.json(list);
-    console.log('Sent list of items');
+const app = express();
+// app.use(bodyParser.urlencoded({ extended: false }));
+const jsonparser = bodyParser.json();
+
+// app.configure(function() {
+//   app.use(sqlinjection); // add sql-injection middleware here
+// });
+
+router.get("/getList", (req, res) => {
+  var list = ["item1", "item2", "item3"];
+  res.json(list);
+  console.log("Sent list of items");
 });
-router.get('/getAllBookings', (req,res) => {
-    queries.getAllBookings()
-    .then(bookings=> bookings.rows)
-    .then(rows=> res.json(rows))
-    .catch(err=> console.log(err));   
+router.get("/getAllBookings", (req, res) => {
+  queries
+    .getAllBookings()
+    .then(bookings => bookings.rows)
+    .then(rows => res.json(rows))
+    .catch(err => console.log(err));
```

Comment:
Would be good to see you forwarding this error to Express Error Handling middleware, rather than just `console.log`-ing it. That way you can send a helpful message back to the client.

Reply:
You don't have to do this now for this PR, but maybe make an issue to create Express Error Handling middleware and forward all server errors here (if you haven't already).

Reply:
oppened a new issue to deal with it properly later